doc/release_notes_0.5.rst

```diff
@@ -0,0 +1,560 @@
+.. _release_notes_0.5:
+
+ACRN v0.5 (Jan 2019)
+####################
+
+We are pleased to announce the release of Project ACRN version 0.5.
+
+ACRN is a flexible, lightweight reference hypervisor, built with
+real-time and safety-criticality in mind, optimized to streamline
+embedded development through an open source platform. Check out the
+:ref:`introduction` for more information.
+
+
+All project ACRN source code is maintained in the
+https://github.com/projectacrn/acrn-hypervisor repository and includes
+folders for the ACRN hypervisor, the ACRN device model, and documentation.
+You can either download this source code as a zip or tar.gz file (see
+the `ACRN v0.5 GitHub release page
+<https://github.com/projectacrn/acrn-hypervisor/releases/tag/v0.5>`_ or
+use Git clone and checkout commands:
+
+.. code-block:: bash
+
+   git clone https://github.com/projectacrn/acrn-hypervisor
+   cd acrn-hypervisor
+   git checkout v0.5
+
+The project's online technical documentation is also tagged to correspond
+with a specific release: generated v0.5 documents can be found at
+https://projectacrn.github.io/0.5/.  Documentation for the latest
+(master) branch is found at https://projectacrn.github.io/latest/.
+
+
+Version 0.5 new features
+************************
+
+**OVMF support initial patches merged in ACRN**: 
+In order to boot Guest Windows on ACRN, the virtual machines firmware is 
+suggested to use OVMF, which is Opensource Virtual Machine Firmware. 
+Initial patches to support OVMF have been merged in ACRN hypervisor. 
+More patches for ACRN and patches upstreaming to OVMF work will be continuing.
+
+**UP2 board serial port support**: 
+Enabled UP2 board’s serial port debugging during SOS and UOS boot.
+One E2E binary to support all UEFI platform: ACRN can support 
+ApolloLake NUC and Kabylake NUC. It’ll be easier to use if 
+we can have an all-in-one E2E build to support all UEFI hardware platforms. 
+From this release, community developers can use one end-to-end reference 
+build to run on both APL NUC and KBL NUC, only need a boot parameter 
+change in `Getting Startup Guide <https://projectacrn.github.io/latest/getting-started/index.html>`_
+
+**APL UP2 board with SBL firmware**: ACRN can support APL UP2 board with 
+Slim Bootloader firmware now from this release. Slim Bootloader is a modern, 
+flexible, light-weither, open source reference boot loader with key benefits 
+like, fast, small, customizable, secure. An end-to-end reference build with 
+ACRN hypervisor, Clear Linux as SOS, Clear Linux as UOS has been tried and 
+verified on UP2/SBL board. A step-by-step Getting Startup Guide (GSG) has 
+been posted in ACRN documentation website.
+
+**Document update**: Several documents have been added or updated in this release. 
+“GPU passthrough guidance on Skylake NUC”, “acrn-dm parameters introduction”. 
+One page ACRN project brochure has been uplifted and able to download.
```

Comment:
Change to:
```
**Document updates**: Several new documents have been added in this release, including:

* :ref:`skl-nuc-gpu-passthrough`
* :ref:`acrn-dm_parameters`
```

Where is this ACRN project brochure uploaded?

Reply:
ACRN-dm parameter updated in Issue 2036,